core/spec.md

```diff
@@ -7193,6 +7261,15 @@ something unexpected happened in the server that caused an error condition.
 * Data: {URL to entity being processed}
 * Detail: {information specific to the processing details}
 
+#### unsupported_sort_attribute
+
+* Type: `https://github.com/xregistry/spec/blob/main/core/spec.md#unsupported_sort_attribute`
+* Code: `400 Bad Request`
+* Instance: The request URL
+* Title: `The specified "sort" attribute is not supported`
```

Comment:
Perhaps "Sorting on the specified attribute is not supported"? It's not that the attribute itself isn't generally supported, it's that you can't sort on it. (At least, that's what I *think* is meant.)

As a thought, is this likely to be due to the type, or the specific attribute? (e.g. because it's not indexed, even though it could be?)

Reply:
I like that phrasing better - thanks.

It's a bit vague as to why this might happen because some implementation choices might find it hard to sort on data that's handled differently from others. For example, the "count" attributes, at least in my impl, are not stored in the DB and are only calculated during serialization due to "filtering". So sorting on them would be a real challenge for me.